synapse/replication/tcp/resource.py

```diff
@@ -159,6 +173,23 @@ async def _run_notifier_loop(self):
                             )
                             stream_updates_counter.labels(stream.NAME).inc(len(updates))
 
+                        else:
+                            # The token has advanced but there is no data to
+                            # send, so we send a `POSITION` to inform other
+                            # workers of the updated position.
+                            logger.info(
+                                "Sending position: %s -> %s", stream.NAME, current_token
+                            )
```

Comment:
Not something for this PR, but I wonder whether these at INFO are a bit noisy.

Reply:
Yeah, probably, its just really useful to figure out when things go wrong.